### DISCUSSION TITLE
https://qudt.org/ website reporting tomcat template.

### DISCUSSION THREAD

**Discussion by oldskeptic** [qudt/qudt-public-repo#486] [Q&A] [answered]
I'm not sure whom to report this to, but the website is returning the tomcat installation template.

"If you're seeing this, you've successfully installed Tomcat. Congratulations!"

**Comment by steveraysteveray** [comment, marked as answer]
You came to the right place. We are in the process of upgrading our entire web page generation approach, but in the meantime, we are hindered by a lack of SSL certificates (long story). So http://qudt.org works, but https://qudt.org just gives you the Tomcat page. Please stick with http://qudt.org for the time being!

**Comment by Maximvdw** [comment]
This will definitely be a big help for link traversal querying in the browser (as traversing qudt via http is not possible in a secure context due to mixed-content violations)

+1

**Reply by steveraysteveray** [reply]
This was all fixed in [Release 2.1.18](https://github.com/qudt/qudt-public-repo/releases/tag/v2.1.18).
